docs/ldml/tr35-personNames.md

```diff
@@ -48,37 +48,37 @@ The LDML specification is divided into the following parts:
-* 2 [XML Structure](#2-xml-structure)
-  * 2.1 [personNames Element](#2-1-personnames-element)
-  * 2.2 [personName Element](#2-2-personname-element)
-  * 2.3 [nameOrderLocales Element](#2-3-nameorderlocales-element)
-  * 2.4 [foreignSpaceReplacement Element](#2-4-foreignspacereplacement-element)
-  * 2.5 [initialPattern Element](#2-5-initialpattern-element)
+* 2 [XML Structure](#xml-structure)
+  * 2.1 [personNames Element](#personnames-element)
+  * 2.2 [personName Element](#personname-element)
+  * 2.3 [nameOrderLocales Element](#nameorderlocales-element)
+  * 2.4 [foreignSpaceReplacement Element](#foreignspacereplacement-element)
+  * 2.5 [initialPattern Element](#initialpattern-element)
     * 2.5.1 [Syntax](#syntax)
-* 3 [Person Name Object](#3-person-name-object)
+* 3 [Person Name Object](#person-name-object)
 * 4 [Person Name Attributes](#4-person-name-attributes)
-  * 4.1 [order](#4-1-order)
-  * 4.2 [length](#4-2-length)
-  * 4.3 [usage](#4-3-usage)
-  * 4.4 [formality](#4-4-formality)
+  * 4.1 [order](#order)
+  * 4.2 [length](#length)
+  * 4.3 [usage](#usage)
+  * 4.4 [formality](#formality)
 * 5 [namePattern Syntax](#5-namepattern-syntax)
-  * 5.1 [Fields](#5-1-fields)
-  * 5.2 [Modifiers](#5-2-modifiers)
-* 6 [Formatting Process](#6-formatting-process)
-  * 6.1 [Derive the name locale](#6-1-derive-the-name-locale)
-  * 6.2 [Derive the formatting locale](#6-2-derive-the-formatting-locale)
-  * 6.3 [Derive the name order](#6-3-derive-the-name-order)
-  * 6.4 [Choose a personName](#6-4-choose-a-personname)
-  * 6.5 [Choose a namePattern](#6-5-choose-a-namepattern)
-  * 6.6 [Examples of choosing a namePattern](#6-6-examples-of-choosing-a-namepattern)
+  * 5.1 [Fields](#fields)
+  * 5.2 [Modifiers](#modifiers)
+* 6 [Formatting Process](#formatting-process)
+  * 6.1 [Derive the name locale](#derive-the-name-locale)
+  * 6.2 [Derive the formatting locale](#derive-the-formatting-locale)
+  * 6.3 [Derive the name order](#derive-the-name-order)
+  * 6.4 [Choose a personName](#choose-a-personname)
+  * 6.5 [Choose a namePattern](#choose-a-namepattern)
+  * 6.6 [Examples of choosing a namePattern](#examples-of-choosing-a-namepattern)
     * 6.6.1 [Examples for rules 1 and 2](#examples-for-rules-1-and-2)
     * 6.6.2 [Examples for rule 3 and the interaction between the rules](#examples-for-rule-3-and-the-interaction-between-the-rules)
-  * 6.7 [Deriving initials](#6-7-deriving-initials)
-  * 6.8 [Handling foreign names](#6-8-handling-foreign-names)
-* 7 [Sample Name](#7-sample-name)
-  * 7.1 [Syntax](#7-1-syntax)
-  * 7.2 [Expected values](#7-2-expected-values)
-* 8 [PersonName Data Interface Examples](#8-personname-data-interface-examples)
+  * 6.7 [Deriving initials](#deriving-initials)
+  * 6.8 [Handling foreign names](#handling-foreign-names)
+* 7 [Sample Name](#sample-name)
+  * 7.1 [Syntax](#syntax)
+  * 7.2 [Expected values](#expected-values)
+* 8 [PersonName Data Interface Examples](#personname-data-interface-examples)
```

Comment:
@macchiati  it looks like the ToC was perhaps updated 'by hand' when it should have been regenerated using the script, I'l open a PR

Reply:
I didn't realize it was generated. That would have saved time trying (and failing) to keep it updated.